.ci/run.sh

```diff
@@ -65,6 +65,10 @@ case "${CI_JOB}" in
 		echo "INFO: Running Confidential Container tests"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-containerd"
 		;;
+	"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
+		echo "INFO: Running Confidential Container tests"
```

Comment:
This is in keeping with the existing code but we should switch to the `info()` call at some point as it's already available to this script.

Reply:
Thank you - we've used the `info()` call instead for line 69! Definitely something that can be applied to the rest of the file at some point tho 👍